app/controllers/idv/review_controller.rb

```diff
@@ -62,9 +62,7 @@ def step_indicator_steps
     end
 
     def flash_message_content
-      if idv_session.address_verification_mechanism == 'gpo'
-        t('idv.messages.mail_sent')
-      else
+      unless idv_session.address_verification_mechanism == 'gpo'
```

Comment:
personal preference for `if !=` but if rubocop requires this, so be it

```suggestion
      if idv_session.address_verification_mechanism != 'gpo'
```

Reply:
I agree, and I'd probably favor removing that rule if it gets in the way